File: sidekick/App/App/Plugins/DashboardPlugin.m
//
//  DashboardPlugin.m
//  App
//
//  Created by Patrick Baxter on 29/11/22.
//

#import <Capacitor/Capacitor.h>

CAP_PLUGIN(DashboardPlugin, "Dashboard",
    CAP_PLUGIN_METHOD(getTest, CAPPluginReturnPromise);
)

File: sidekick/App/App/Plugins/DashboardPlugin.swift
//
//  DashboardPlugin.swift
//  App
//
//  Created by Patrick Baxter on 29/11/22.
//

import Capacitor
import shared

@objc(DashboardPlugin)
public class DashboardPlugin: CAPPlugin {
    @objc func getTest(_ call: CAPPluginCall) {
        call.resolve(["test": Dashboard().test()])
    }
}
